setup.cfg

```diff
@@ -20,3 +20,5 @@ packages =
 [entry_points]
 glance_store.drivers =
     bbchttp = bbc_openstack_plugins.glance_store._drivers.http:Store
+nova.scheduler.filters =
+    CustomDiskFilter = bbc_openstack_plugins.nova.scheduler.filters.custom_disk_filter:CustomDiskFilter
```

Comment:
We should change this name to something namespaced for our product. "Custom" is too generic.

Maybe "BBCDiskFilter"?